Applications/Spire/Include/Spire/Ui/DropDownList.hpp

```diff
@@ -58,6 +58,9 @@ namespace Spire {
       //! no item is currently activated.
       void activate_previous();
 
+      //! Removes the highlight from the activated item.
+      void clear_activated_item();
```

Comment:
"activated" is a verb, the noun is "active", hence rename this to "clear_active_item".